package.json

```diff
@@ -20,6 +20,7 @@
   "main": "index.js",
   "devDependencies": {
     "@bcgov/bc-sans": "^1.0.1",
+    "@bcgov/webpack-wordpress": "git+https://apps.itsm.gov.bc.ca/bitbucket/scm/wp/bcgov-webpack-wordpress.git",
```

Comment:
It might be a good idea to attach a version specific to the weback-wordpress script

Reply:
Looks like we can add version requirements to the end with a # (https://docs.npmjs.com/cli/v8/configuring-npm/package-json#git-urls-as-dependencies) so I'll see about doing that in the next PR.